text/0028-indexing-csv.md

```diff
@@ -52,14 +52,15 @@ While there's [RFC 4180](https://tools.ietf.org/html/rfc4180) as a try to add a
 ##### `null` value
 
 - If a field is of type `string`, then an empty cell is considered as a `null` value (e.g. `,,`), anything else is turned into a string value (e.g. `, ,` is a single whitespace string)
-- If a field is of type `number`, when the trimmed field is empty, it's considered as a `null` value (e.g. `,,` `, ,`); otherwise Meilisearch try to parse the number.
+- If a field is of type `boolean`, when the trimmed field is empty, it's considered as a `null` value (e.g. `,,` `, ,`); otherwise, Meilisearch tries to parse the number.
+- If a field is of type `number`, when the trimmed field is empty, it's considered as a `null` value (e.g. `,,` `, ,`); otherwise, Meilisearch tries to parse the boolean as either `true` or `false`.
```

Comment:
It seems that the two ends of the sentences have been inverted by mistake. It wouldn't have happened with **Emacs** according to my Jedi senses 🥁 ✨

Reply:
Oops, yes, good catch! Sorry it took me 16h to fix it; it’s because I had to configure emacs